src/components/Schematics/Schematic-Node.vue

```diff
@@ -204,6 +223,21 @@ export default {
         </v-avatar>
         {{ nodeData.data.name }}
       </div>
+
+      <div v-if="mappedChildren" :style="durationStyle">
+        <div v-if="showDetails">
+          {{
+            mappedChildren.state_counts['Success']
+              ? mappedChildren.state_counts['Success']
+              : 0
+          }}/{{ totalRuns }} {{ totalRuns > 1 ? 'runs' : 'run' }} complete
+        </div>
+
+        <div v-else class="mt-2">
+          {{ mappedChildren.state_counts['Success'] }}/{{ totalRuns }}
```

Comment:
Don't we want to count all the runs that have entered any finished state instead of just `Success`?